wsd/annotate/__init__.py

```diff
@@ -1,2 +1,2 @@
 """A collection of mesop components to use in the annotation ui."""
-from .components import *
+# from .components import *
```

Comment:
We need this import here. I get 

```
  File "/Users/arnaud/workspace/linalgo/wsd/wsd/annotate/app.py", line 11, in <module>
    from wsd.annotate import Token, lin_doc, lin_entry
ImportError: cannot import name 'Token' from 'wsd.annotate'
```

otherwise.